README.md

```diff
@@ -65,12 +65,22 @@ Need help? This is a developer's portal for CoBlocks and should not be used for
 
 ## Contributions
 
-Please read the [guidelines for contributing](https://github.com/godaddy/coblocks/blob/master/.github/CONTRIBUTING.md) to CoBlocks. Anyone is welcome to contribute. If you find a 🐞 or an issue, [create an issue](https://github.com/godaddy/coblocks/issues/new). Here are ways to contribute:
+Thank you for thinking about contributing to CoBlocks plugin! If you're unsure of anything, know that you're 💯 welcome to submit an issue or pull request on any topic. The worst that can happen is that you'll be politely directed to the best location to ask your question or to change something in your pull request. We appreciate any sort of contribution and don't want a wall of rules to get in the way of that.
```

Comment:
Maybe we update this to "Thank you for considering contributing to CoBlocks!"

and "We appreciate any and all contributions and don't want a wall of rules to get in the way of that."

Reply:
> "Thank you for considering contributing to CoBlocks!"
Too many 'c' sounds, alliteration.

> "Thank you for considering contributing to CoBlocks!"
Agree with this one.